code/languages/com.mbeddr.build/solutions/com.mbeddr.platform/models/com/mbeddr/platform/build.mps

```diff
@@ -247,6 +247,7 @@
   <node concept="1l3spW" id="3AVJcIMlF8l">
     <property role="TrG5h" value="com.mbeddr.platform" />
     <property role="2DA0ip" value="../../../../../build/com.mbeddr.platform" />
+    <uo k="s:LocalSourcePathArtifact" v="i:23468bc7" />
```

Comment:
Looks like some user objects were stored in the model? Where do they come from? Seems wrong.

Reply:
Nice catch, I have fixed the build script but I don't know what happened.